packages/ui/src/utils/__tests__/ExitTransitionPlaceholder.spec.tsx

```diff
@@ -0,0 +1,27 @@
+import { render, screen } from '@testing-library/react';
+import { ExitTransitionPlaceholder } from '../ExitTransitionPlaceholder';
+
+it('render null child', () => {
+  const child = null;
+  const { container } = render(
+    <ExitTransitionPlaceholder in={true}>{child}</ExitTransitionPlaceholder>,
+  );
+  expect(container).toBeEmptyDOMElement();
+});
+
+it('render null with boolean child', () => {
+  const child = true;
+  const { container } = render(
+    <ExitTransitionPlaceholder in={true}>{child}</ExitTransitionPlaceholder>,
+  );
+  expect(container).toBeEmptyDOMElement();
+});
+
+it('render child', () => {
+  render(
+    <ExitTransitionPlaceholder in={true}>
+      <div>Test</div>
+    </ExitTransitionPlaceholder>,
+  );
+  expect(screen.getByText('Test')).toBeInTheDocument();
+});
```

Comment:
@tokarevfedor1991 Can we add case where previous child is different with next child when `in` prop changes?

You can do by using rerender method from result https://testing-library.com/docs/react-testing-library/api#rerender

Reply:
ok, I will add

Reply:
@doniyor2109  It's done